### PR TITLE
autojump: various tweaks

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

File: pages/common/autojump.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-> Automatically jump to the directory you visited the most. Aliases like j or jc can also be used for fewer typing.
+> Quickly jump among the directories you visit the most.
+> Aliases like j or jc are provided for even less typing.
 
 - Jump to a directory that contains the given pattern.
 
 `j {{pattern}}`
 
-- Jump to a sub-directory of current directory that contains the given pattern.
+- Jump to a sub-directory (child) of the current directory that contains the given pattern.
```
Comment (igorshubovych) [inline pages/common/autojump.md]: I don't understand why you added (child) here. Sub-directory is a common, very simple for understanding term.

Comment (waldyrious) [inline pages/common/autojump.md]: It's a mnemonic for the "c" in "jc".

Comment (igorshubovych) [inline pages/common/autojump.md]: Oh. Then it is good.

**Comment by igorshubovych** [comment]
LGTM

**Comment by felixonmars** [comment]
:+1:

**Pull request merged**